docs/rfc/0002_kubectl_rw.md

```diff
@@ -0,0 +1,49 @@
+|                    |                                      |
+| -------            |--------------------------------------|
+| Feature            | kubectl-rw: a kind of kubectl plugin |
+| Status             | In-progress                          |
+| Date               | 2022-06-29                           |
+| Authors            | Xinyu                                |
+| RFC PR #           |                                      |
+| Implementation PR #|                                      |
+|                    |                                      |
+
+# **Summary**
+
+kubectl-rw: a kubectl-plugin to deploy and manage risingwave operator and DBs.
+
+# **Motivation**
+
+For convenience, we will design a command line tool that allows users to quickly manage the risingwave operator and DBs.
+
+For example, user can do the operations as follows:
+
+```shell
+
+kubectl rw init # deploy the operator in the kubernetes cluster
+
+kubectl rw create instance --meta-storage memory # create a risingwave db in the kubernetes cluster and use the memory storage for mete-node
+
+kubectl rw upgrade # upgrade the risingwave db to the latest version
+```
+### What commands will be supported (Continually updated)
+
+- init: deploy the operator.
+- create: create the risingwave db.
+- delete: delete the risingwave db.
+- upgrade: upgrade the risingwave db.
+- scale: scale the risingwave db.
+
```

Comment:
shall we also have `update`?

Reply:
That's exactly how I feel. We need something like `restart`/`stop`/`resume`/`scale`/`update`.

Reply:
And we also need these two commands:
- `get` for getting the detailed and human-readable status of the RisingWave cluster.
  - reporting running status, connection info, and other things might be helpful for humans (developers/users)
- `list` for listing the RisingWave clusters in the cluster.

Reply:
Maybe in the future, we need something like `config` to update the configurations of the RisingWave cluster on the fly. There's a lot of room for imagination.